test/server/data/page/src.html

```diff
@@ -60,6 +60,11 @@
         <!--<script type="text/javascript">-->
         var someScript;
     </script>
+    <script type="module">
```

Comment:
Support the `import.meta.url` property

```js
<html>
    <head>
		<script type="module">
			console.log(import.meta.url); // url of the html pag on which the module was included
		</script>
	</head>
	<body>
	</body>
</html>
```

Reply:
https://github.com/DevExpress/testcafe-hammerhead/issues/2118